AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Main_Null_Render_MaterialEditor_01.py

```diff
@@ -0,0 +1,25 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+import logging
+import pytest
+
+from ly_test_tools.o3de.material_editor_test import MaterialEditorBatchedTest, MaterialEditorTestSuite
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+@pytest.mark.parametrize("launcher_platform", ['windows_material_editor'])
+class TestMaterialEditor(MaterialEditorTestSuite):
+
+    class MaterialEditor_Atom_LaunchMaterialEditor_1(MaterialEditorBatchedTest):
```

Comment:
Automated Review shows this module as failing due to collecting zero tests. Perhaps due to these inner classes not containing 'test' in their name? Should be locally reproducible by running the following command from the local CMake build directory:
`ctest -C profile -R AutomatedTesting::Atom_Main_Null_Render_MaterialEditor.main --output-on-failure`

Reply:
was a bug in `CMakeLists.txt` just updated it